pyweek36/game.py

```diff
@@ -295,12 +305,15 @@ def on_draw(self):
             self.bullet_list.draw()
             self.player_sprite.draw()
         else:
-            self.textures["death_animation"].draw_scaled(
-                self.width / 2 + self.camera.position.x,
-                self.height / 2 + self.camera.position.y,
-                DEATH_ANIMATION_SCALE
-                * math.sin(
-                    (math.pi / 4)
-                    * (DEATH_ANIMATION_TIME - (self.global_time - self.dead))
-                ),
-            )
+            try:
+                self.textures["death_animation"].draw_scaled(
+                    self.width / 2 + self.camera.position.x,
+                    self.height / 2 + self.camera.position.y,
+                    DEATH_ANIMATION_SCALE
+                    * math.sin(
+                        (math.pi / 4)
+                        * (DEATH_ANIMATION_TIME - (self.global_time - self.dead))
+                    ),
+                )
+            except:
```

Comment:
Bare except should be avoided. I imagine it would be for domain error in `math.sin`?

Reply:
It's a weird pyglet state error or something, I opened #50